CSS grid layout guide refresh: auto-placement and alignment (#38004)
mdn/content: files/en-us/web/css/css_grid_layout/box_alignment_in_grid_layout/index.md

{{ EmbedLiveSample('Setting_align-content_space-between', '500', '600') }}

> If an item spans more than one grid track, using a space distribution value will likely cause items on your grid to become larger as the space that is added between the tracks gets added to the spanning item. Therefore, if you use these values, ensure the content of your tracks can cope with the extra space, or that you have used alignment properties on the items, to cause them to move to the start or end rather than stretch.

Reviewer: What do you think?

```suggestion
If an item spans more than one grid track, using a space distribution value will likely cause items on your grid to become larger because the space that's added between the tracks is added to the spanning item. Therefore, if you use these values, make sure the content of the tracks can cope with the extra space or that you have used alignment properties on the items, so they move to the start or end rather than stretch.
```
